core/lib/refinery/core/configuration.rb

```diff
@@ -68,12 +58,8 @@ def clear_stylesheets!
         self.stylesheets = []
       end
 
-      def dragonfly_custom_backend?
-        config.dragonfly_custom_backend_class.present?
-      end
-
       def dragonfly_custom_backend_class
-        config.dragonfly_custom_backend_class.constantize if dragonfly_custom_backend?
+        raise "Refinery::Dragonfly now handles all dragonfly configuration. Consult 'config/initializers/refinery/dragonfly.rb'."
```

Comment:
You should deprecate this method instead of raising an error.

Reply:
Right now this *is* refinerycms-dragonfly.  Next release we can split it into a separate gem (refinerycms-dragonfly) and an uploader and a generic image handler.

I absolutely agree that we should move further away from dragonfly - this is just one step.  And once it is out there working, creating it as an external gem is easy.

To move away from Dragonfly altogether we need to think about how we handle image resizing and conversion - at the moment it is all done using ImageMagick geometry. We probably need to abstract this enough to handle other graphics solutions (Cloudinary for instance).

Comments about repositories etc all fine.

Reply:
@anitagraham I agree with you too, could you update your PR with the minor changes i requested?